Too-wide version bounds in commonmark-extensions

I just tried building a new Hakyll project, and got the following error from `commonmark-extensions`:
```
src/Commonmark/Extensions/FancyList.hs:32:17: error:
    Variable not in scope:
      getParentListType
        :: ParsecT [Tok] (BPState m il bl) m (Maybe ListType)
   |
32 |   mbListType <- getParentListType
   |                 ^^^^^^^^^^^^^^^^^
```

The issue seems to be that `commonmark-extensions` version `0.2.3.1` specifies a lower bound of `0.2` for `commonmark`, but `Commonmark.Blocks` in that package didn't export `getParentListType` until version `0.2.2`.

--- a/commonmark-extensions/commonmark-extensions.cabal
+++ b/commonmark-extensions/commonmark-extensions.cabal
@@ -59,7 +59,7 @@ library
     , transformers
     , filepath
     , network-uri
-    , commonmark >= 0.2 && < 0.3
+    , commonmark >= 0.2.2 && < 0.3
     -- for extensions:
     , emojis >= 0.1 && < 0.2
   exposed-modules:

